modules/bound-sa-tokens-configuring.adoc

```diff
@@ -122,8 +122,13 @@ spec:
 ----
 <1> A reference to an existing service account.
 <2> The path relative to the mount point of the file to project the token into.
-<3> Optionally set the expiration of the service account token, in seconds. The default is 3600 seconds (1 hour) and must be at least 600 seconds (10 minutes). The kubelet will start trying to rotate the token if the token is older than 80 percent of its time to live or if the token is older than 24 hours.
+<3> Optionally set the expiration of the service account token, in seconds. The default is 3600 seconds (1 hour) and must be at least 600 seconds (10 minutes). The kubelet will start trying to rotate the token if the token is older than 80 percent of its time to live or if the token is older than 24 hours. 
 <4> Optionally set the intended audience of the token. The recipient of a token should verify that the recipient identity matches the audience claim of the token, and should otherwise reject the token. The audience defaults to the identifier of the API server.
++
+[NOTE]
+====
+Currently, {product-title} overrides the `expirationSeconds` value for one year from the initial token generation with the `--service-account-extend-token-expiration` default of `true`. This is to prevent unexpected failure. Users cannot change this setting.
```

Comment:
`Currently` is not required since the documentation refers to the current working OCP. Merged the sentence to avoid the ambiguous usage of pronoun 'This'. 
The sentence somehow conveys that it `overrides for one year`. Does it mean the overridden value is set to true for a year? May be you can try rewording this bit if IIUC.

```suggestion
In order to prevent unexpected failure, {product-title} overrides the `expirationSeconds` value for one year from the initial token generation with the `--service-account-extend-token-expiration` default of `true`. You cannot change this setting.
```

Reply:
I like those changes, thank you. I clarified that it is overridden to be one year instead of being overridden for one year.